### PR TITLE
Feature/login redesign

### PR DESCRIPTION
**Pull request by vafomin**
- Redesign login page
- Redesign invite page
- Rewrite login test
- New screen for Invalid error

### CODE REVIEW

File: packages/asc-web-components/code-input/code-input.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { mount } from "enzyme";
+import CodeInput from ".";
+
+describe("<CodeInput />", () => {
+  it("renders without error", () => {
```
Comment (AlexeySafronov) [inline packages/asc-web-components/code-input/code-input.test.js:6]: Need more tests on id, className and style

File: web/ASC.Web.Login/src/Login.jsx [hunk 2]
```diff
@@ -157,11 +159,14 @@ const Form = (props) => {
     error && setErrorText(error);
     confirmedEmail && setIdentifier(confirmedEmail);
 
+    await setProviders();
+    await getSso();
```
Comment (AlexeySafronov) [inline web/ASC.Web.Login/src/Login.jsx:163]: Combine with Promis.all

File: web/ASC.Web.Login/src/RoomsLogin.jsx [hunk 3]
```diff
@@ -140,11 +142,14 @@ const Form = (props) => {
     error && setErrorText(error);
     confirmedEmail && setIdentifier(confirmedEmail);
 
+    await setProviders();
+    await getSso();
```
Comment (AlexeySafronov) [inline web/ASC.Web.Login/src/RoomsLogin.jsx:146]: Combine with Promis.all

File: web/ASC.Web.Client/src/Shell.jsx [hunk 4]
```diff
@@ -417,7 +446,9 @@ const Shell = ({ items = [], page = "home", ...rest }) => {
               <PublicRoute exact path={WIZARD_URL} component={WizardRoute} />
               <PrivateRoute path={ABOUT_URL} component={AboutRoute} />
               {loginRoutes}
+              <Route path={ENTER_CODE_URL} component={EnterCodeRoute} />
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/Shell.jsx:449]: Route must be hidden if roomsMode == false

Comment (vafomin) [reply web/ASC.Web.Client/src/Shell.jsx]: Resolve: https://github.com/ONLYOFFICE/AppServer/pull/517/commits/da121a1bfeef2311768464a0929cfbdac7b6ad98

File: web/ASC.Web.Login/src/RoomsLogin.jsx (hunk 3 above)
Comment (vafomin) [reply web/ASC.Web.Login/src/RoomsLogin.jsx]: Resolve: https://github.com/ONLYOFFICE/AppServer/pull/517/commits/8bba8e834592e6087554131a13adc36246ad8f74

File: web/ASC.Web.Login/src/Login.jsx (hunk 2 above)
Comment (vafomin) [reply web/ASC.Web.Login/src/Login.jsx]: Rsolve: https://github.com/ONLYOFFICE/AppServer/pull/517/commits/8bba8e834592e6087554131a13adc36246ad8f74

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**